config/sync/core.entity_form_display.paragraph.info_box.default.yml

```diff
@@ -5,7 +5,7 @@ dependencies:
   config:
     - field.field.paragraph.info_box.field_content_block_text
     - field.field.paragraph.info_box.field_paragraph_title
-    - field.field.paragraph.info_box.field_variant
+    - field.field.paragraph.info_box.field_variant_select
```

Comment:
It may be worth not including `_select` in the field machine name. It's not important (to not include it), but we usually don't embed the field type in the name.

Renaming the machine name can be done with search and replace in the config files and by renaming files with `field_variant_select` in the name.

Reply:
Absolutely - Changed with latest commit.